src/cunumeric/sort/sort.cu

```diff
@@ -30,6 +31,11 @@
 
 #include "cunumeric/cuda_help.h"
 
+// above this threshold segment sort will be performed
+// by cub::DeviceSegmentedRadixSort instead of thrust::(stable_)sort
+// with tuple keys (not available for complex)
+#define SEGMENT_THRESHOLD_RADIX_SORT 400
```

Comment:
it's fine to keep this as a macro for now, but I feel we will later want to be able to configure this via a shell variable.